src/app/screens/auth/auth-routing.module.ts

```diff
@@ -0,0 +1,22 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { LoginpageComponent } from './components/loginpage/loginpage.component';
+import { RegustrationComponent } from './components/regustration/regustration.component';
+
+const routes: Routes = [
+  {
+    path: 'login',
+    component: LoginpageComponent
+  },
+  {
+    path: 'Registration',
+    component: RegustrationComponent,
```

Comment:
Spelling mistake. Good job to explore multiple routes within feature module

Reply:
Thank you sir